### PR TITLE
add new users to the latest edition

### PR DESCRIPTION
**Pull request by StefVdHaute**

### CODE REVIEW

File: backend/app/routers/editions.py [hunk 1]
```diff
@@ -65,6 +65,13 @@ async def get_editions(session: AsyncSession = Depends(get_session), role: RoleC
         return [results[0].uri] if results else []
 
 
+@router.get("/current_edition", response_description="Edition retrieved")
+async def get_current_edition(session: AsyncSession = Depends(get_session)):
+    result = (await session.execute(select(Edition.year))).all()
+    year = max((row[0] for row in result))
+    return await read_where(Edition, Edition.year == year, session=session)
```
Comment (QuintenVervynck) [inline backend/app/routers/editions.py:72]: you query the db twice while you could do it in one query + sorting like we did on the other places we needed the current edition.

What is the advantage of doing it like this?

**Review by QuintenVervynck — CHANGES_REQUESTED** [review]

File: backend/app/routers/editions.py (hunk 1 above)
Comment (StefVdHaute) [reply backend/app/routers/editions.py:72]: There is no advantage i just didn't think of it, i'll change it

**Review by QuintenVervynck — APPROVED** [review]
looks good now!  fix that flake8 and merge

**Comment by QuintenVervynck** [comment]
owh it's gone already

**Comment by QuintenVervynck** [comment]
tests do fail but I'll leave that up to you if you want to merge or not

**Comment by StefVdHaute** [comment]
I don't know why the tests fail, I'll look into that but locally they worked.

**Pull request merged**